src/pudl/etl/glue_assets.py

```diff
@@ -42,28 +46,609 @@ def create_glue_tables(
         eia=dataset_settings.glue.eia,
     )
 
-    # Add the EPA to EIA crosswalk, but only if the eia data is being processed.
-    # Otherwise the foreign key references will have nothing to point at:
-    ds = context.resources.datastore
-    if dataset_settings.glue.eia:
-        # Check to see whether the settings file indicates the processing of all
-        # available EIA years.
-        processing_all_eia_years = (
-            dataset_settings.eia.eia860.years
-            == dataset_settings.eia.eia860.data_source.working_partitions["years"]
-        )
-        glue_raw_dfs = pudl.glue.epacamd_eia.extract(ds)
-        glue_transformed_dfs = pudl.glue.epacamd_eia.transform(
-            glue_raw_dfs,
-            generators_entity_eia,
-            boilers_entity_eia,
-            processing_all_eia_years,
-        )
-        glue_dfs.update(glue_transformed_dfs)
-
     # Ensure they are sorted so they match up with the asset outs
     glue_dfs = dict(sorted(glue_dfs.items()))
 
     return (
         Output(output_name=table_name, value=df) for table_name, df in glue_dfs.items()
     )
+
+
+#####################
+# EPA - EIA Crosswalk
+#####################
+
+
+@asset(required_resource_keys={"datastore"})
+def raw_epacamd_eia(context) -> pd.DataFrame:
+    """Extract the EPACAMD-EIA Crosswalk from the Datastore."""
+    logger.info("Extracting the EPACAMD-EIA crosswalk from Zenodo")
+    ds = context.resources.datastore
+    with ds.get_zipfile_resource("epacamd_eia", name="epacamd_eia.zip").open(
+        "camd-eia-crosswalk-master/epa_eia_crosswalk.csv"
+    ) as f:
+        return pd.read_csv(f)
+
+
+@asset(
+    required_resource_keys={"dataset_settings"}, io_manager_key="pudl_sqlite_io_manager"
+)
+def clean_epacamd_eia(
+    context,
+    raw_epacamd_eia: pd.DataFrame,
+    generators_entity_eia: pd.DataFrame,
+    boilers_entity_eia: pd.DataFrame,
+) -> pd.DataFrame:
+    """Clean up the EPACAMD-EIA Crosswalk file.
```

Comment:
I am now second-guessing on this asset name. If this lil guy is loaded into the db - which I believe it should be bc it is distinct from the new derived subplant_id table.... should it just be `epacamd_eia`?

Reply:
Yeah, I think it should be named `epacamd_eia` because it will be in the db. For now, the `raw_` and `clean_` prefixes are used for interim assets that aren't included in the db.

Reply:
is that not what the OG crosswalk is called though?

Reply:
this is the og crosswalk!

Reply:
and okay ty this is helpful! will rename back to `epacamd_eia`